### ISSUE TITLE
[NetEdit] After selecting multiple edges, lane addition adds only a lane to one edge.

### ISSUE THREAD

**Issue by bcoueraud87** [eclipse/sumo#12066]
In NetEdit, create a network with two edges. Then with the help of the select mode, select these edges and right click on the first edge in order to add a lane. Then, a new lane will appear for the first edge, but not the second, although it was selected.

**Comment by namdre** [comment]
the lane operation 'duplicate' operates on selected lanes rather than selected edges. If you select a lane on each edge (shift click in select mode) the operation will add a lane to each edge. (Also if you select all lanes of a single edge, and perform 'duplicate', the number of lanes will effectively double).

This is relevant because all properties of the lane are duplicated and the lane properties may vary from on edge to the next.

To duplicate the rightmost lane on all edges you could do the following: 
1. toggle selection to work on lanes by default (Alt+5)
2. select the desired lanes
3. apply duplication

or:
1. select the desired edges
2. in select model click: 'select children'
3. toggle selection modifcation mode to 'keep'
4. select lanes by attribute index=0
5. apply duplication

or (for edges with the same number of lanes):
1. select edges
2. use inspect mode to set a new lane number for all

or if you want to add a restricited lane (sidewalk, bikelane,. ...)
1. select edges
1. use lane operation 'add restricted lane'  

In short i'd say this is working as intended.
